pkg/test/environment.go

```diff
@@ -92,7 +92,7 @@ func NewEnvironment(scheme *runtime.Scheme, options ...functional.Option[Environ
 	if version.Minor() >= 24 {
 		// MinDomainsInPodTopologySpread enforces a minimum number of eligible node domains for pod scheduling
 		// See https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#spread-constraint-definition
-		// Ref: https://github.com/aws/karpenter-core/pull/330
+		// Ref: https://sigs.k8s.io/karpenter/pull/330
```

Comment:
This looks wrong, but we should check if any other links like this exist and make sure that we are routing to the correct place